app/test_main.py

```diff
@@ -1,3 +1,18 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_equal_right_result():
+    args = (
```

Comment:
Please, create separate tests for every case and name them to describe each test case

Reply:
It's better practice to create a separate test per case because it's much easier to find an input that leads to an error if some test failed